### DISCUSSION TITLE
Submit your Practicum exercise (October edition)!

### DISCUSSION THREAD

**Discussion by onlyphantom** [onlyphantom/llm-python#39] [Practicum: Generative AI]
Create a copy of [the Generative AI Workshop template](https://colab.research.google.com/drive/1RNp6mvOI9UGYHYuag76u4erVrnFLGjZS?usp=sharing) (also in this very [GitHub repository (@onlyphantom/llm-python/workshop)](https://github.com/onlyphantom/llm-python/blob/main/workshop/Generative_AI_Template_02.ipynb)).

You need to complete both parts successfully, implementing (1) through (10). 
If you score higher than 8 points (out of a possible 10) you will obtain a certification jointly issued by Supertype and Sectors.

**Run all code and show all output in your Colab notebook before submission.**

You can discuss, and ask for help [in Discord](https://discord.com/invite/TAnZMmNS4X), but please do not share your answers or plagiarize. You are only cheating yourself off an education.

## Submission Format
To submit your completed exercise, open a new Discussion under 'Practicum: Generative AI', and paste a link to the completed work. This can be a .ipynb or a Google Colab notebook. The output **of each cell** must be shown, whether they are intermediary steps or not. Make sure to 'Run All' from top to bottom before saving your work.

Specify your name and first 4 letters of your email at the top of your notebook for me to identify you. 
Example: `Samuel Chan | samu` (if my email is indeed samuel@sectors.app)

> 🔴 IMPORTANT: DO NOT SHARE YOUR SECRETS. USE `from google.colab import userdata` to read your environment variables. Under no circumstance should you secret keys be ever visible. An inclusion of your key in the notebook will automatically disqualify your work, and for your own benefit, **I will promptly remove your thread here on GitHub so that your keys are not stolen**.

**Comment by debrianruhut** [comment]
Debrian Ruhut Saragih | [debrianruhut.saragih@gmail.com](mailto:debrianruhut.saragih@gmail.com)
Link to Google Colab : https://colab.research.google.com/drive/1oTYbSavS9Ym5ww-LZzxquXn3t-SDGto6?usp=sharing

**Comment by BudiRustandi** [comment]
Budi Rustandi | b_ru

Python Notebook in Google Drive link: https://drive.google.com/file/d/1pkfA9w-JpLslimyxVYs1JYkhL-o3an1Z/view?usp=sharing
